messages/notify.go

```diff
@@ -55,6 +59,15 @@ func (n *Notify) SerializeTo(b []byte) error {
 	n.Header.Payload = make([]byte, n.Len()-8)
 
 	var offset = 0
+
+	if n.Status == nil {
+		return ErrMandatoryNotifyStatusPayloadMissing
+	}
```

Comment:
I'd like to leave this package flexible enough to be able to violate the specification for some purposes like security audit etc, and that's why I haven't implemented this kind of validation in basic functions.
However, I understand the validation is helpful in most of the use cases, so I suggest defining package-wide variable `validationEnabled` and set to true by default, but can be disabled by calling `DisableValidation()` to false manually. Anyway please let it unchecked at this time, and let us consider more carefully how we should approach it.

Reply:
Okay, I understand the policy. I'll leave just a comment about it there.

Reply:
Fixed at 86e89579c0ab1ddd421eba3f684087148b014574

Reply:
Thank you for leaving the comment.
This is so trivial but could you refer RFC4666 instead of RFC3332 which was obsoleted?

Reply:
Yeah, absolutely yes!
Fixed at fbbbb01d19b491158a69086c0027770f7c79d9ca

Reply:
🎉